hypervisor/src/kvm/riscv64/aia.rs

```diff
@@ -19,15 +19,9 @@ pub struct KvmAiaImsics {
     /// AIA APLIC address
     aplic_addr: u64,
 
-    /// AIA APLIC size
-    aplic_size: u64,
-
     /// AIA IMSIC address
     imsic_addr: u64,
 
-    /// AIA IMSIC size
-    imsic_size: u64,
-
     /// Number of CPUs handled by the device
     vcpu_count: u64,
```

Comment:
Could you change this internal type?

Reply:
Well, you never use it as a u64 value so better to switch it to u32 rather than casting later.

Reply:
> Well, you never use it as a u64 value so better to switch it to u32 rather than casting later.

Noted 🫡